src/sqlparse.yrl

```diff
@@ -369,7 +370,7 @@ Rootsymbol sql_list.
 Left        100 OR.
 Left        200 AND.
 Left        300 NOT.
-Nonassoc    400 COMPARISON.
+Nonassoc    400 '=' COMPARISON.

```

Comment:
The problem is the so called assignment rule (not really an assignment since assignment is part of the PL/SQL syntax and here the operator is `:=`) should only allow `=`:

`assignment -> column '=' scalar_opt_as_exp`

from the Oracle documentation:

![image](https://user-images.githubusercontent.com/1956149/32478732-b27ffe3c-c386-11e7-95ef-8209a7a8b934.png)